src/definitions/modules/progress.less

```diff
@@ -185,40 +185,40 @@
   }
 
   /* Single Digits */
-  .ui.indicating.progress[data-percent="1"] .bar,
-  .ui.indicating.progress[data-percent="2"] .bar,
-  .ui.indicating.progress[data-percent="3"] .bar,
-  .ui.indicating.progress[data-percent="4"] .bar,
-  .ui.indicating.progress[data-percent="5"] .bar,
-  .ui.indicating.progress[data-percent="6"] .bar,
-  .ui.indicating.progress[data-percent="7"] .bar,
-  .ui.indicating.progress[data-percent="8"] .bar,
-  .ui.indicating.progress[data-percent="9"] .bar {
+  .ui.indicating.progress[data-percent="1"] .bar, .ui.indicating.progress[data-percent^="1."] .bar,
+  .ui.indicating.progress[data-percent="2"] .bar, .ui.indicating.progress[data-percent^="2."] .bar,
+  .ui.indicating.progress[data-percent="3"] .bar, .ui.indicating.progress[data-percent^="3."] .bar,
+  .ui.indicating.progress[data-percent="4"] .bar, .ui.indicating.progress[data-percent^="4."] .bar,
+  .ui.indicating.progress[data-percent="5"] .bar, .ui.indicating.progress[data-percent^="5."] .bar,
+  .ui.indicating.progress[data-percent="6"] .bar, .ui.indicating.progress[data-percent^="6."] .bar,
+  .ui.indicating.progress[data-percent="7"] .bar, .ui.indicating.progress[data-percent^="7."] .bar,
+  .ui.indicating.progress[data-percent="8"] .bar, .ui.indicating.progress[data-percent^="8."] .bar,
+  .ui.indicating.progress[data-percent="9"] .bar, .ui.indicating.progress[data-percent^="9."] .bar {
     background-color: @indicatingFirstColor;
   }
-  .ui.indicating.progress[data-percent="0"] .label,
-  .ui.indicating.progress[data-percent="1"] .label,
-  .ui.indicating.progress[data-percent="2"] .label,
-  .ui.indicating.progress[data-percent="3"] .label,
-  .ui.indicating.progress[data-percent="4"] .label,
-  .ui.indicating.progress[data-percent="5"] .label,
-  .ui.indicating.progress[data-percent="6"] .label,
-  .ui.indicating.progress[data-percent="7"] .label,
-  .ui.indicating.progress[data-percent="8"] .label,
-  .ui.indicating.progress[data-percent="9"] .label {
+  .ui.indicating.progress[data-percent="0"] .label  .ui.indicating.progress[data-percent^="0."] .label,
```

Comment:
A comma is missing here

Reply:
The good method to solve the comma problem to Github is to commit on this commit or rebase ...

Reply:
If we have the coice, then rebase 😄 .

Reply:
I'm confuse I don't know what to do :
![image](https://user-images.githubusercontent.com/1622751/79751883-a68b8480-8313-11ea-9483-dd3caf032d59.png)

Sorry I'am very null with git :s

Reply:
Some PRs have been merged into develop since you created your branch.
Just merge, it should be fine

Reply:
I rebase locally the two commit (first + comma) with comma commit with squash mode

![image](https://user-images.githubusercontent.com/1622751/79752435-96c07000-8314-11ea-9daf-687d892438b8.png)

but when I push on github I get this previous message

Sorry I'am very null with git :s

Reply:
Click on the "Merge" button ;)

Reply:
I do it, thx @lubber-de